app/src/androidTest/java/org/liberty/android/fantastischmemo/test/ui/HintTest.java

```diff
@@ -0,0 +1,99 @@
+//
+//package org.liberty.android.fantastischmemo.test.ui;
+//
+//
+//import android.support.test.espresso.ViewInteraction;
+//import android.support.test.rule.ActivityTestRule;
+//import android.support.test.runner.AndroidJUnit4;
+//import android.test.suitebuilder.annotation.LargeTest;
+//import android.view.View;
+//import android.view.ViewGroup;
+//import android.view.ViewParent;
+//
+//import org.hamcrest.Description;
+//import org.hamcrest.Matcher;
+//import org.hamcrest.TypeSafeMatcher;
+//import org.junit.Rule;
+//import org.junit.Test;
+//import org.junit.runner.RunWith;
+//import org.liberty.android.fantastischmemo.R;
+//import org.liberty.android.fantastischmemo.ui.AnyMemo;
+//
+//import static android.support.test.espresso.Espresso.onView;
+//import static android.support.test.espresso.action.ViewActions.click;
+//import static android.support.test.espresso.action.ViewActions.scrollTo;
+//import static android.support.test.espresso.assertion.ViewAssertions.matches;
+//import static android.support.test.espresso.contrib.RecyclerViewActions.actionOnItemAtPosition;
+//import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+//import static android.support.test.espresso.matcher.ViewMatchers.withClassName;
+//import static android.support.test.espresso.matcher.ViewMatchers.withContentDescription;
+//import static android.support.test.espresso.matcher.ViewMatchers.withId;
+//import static android.support.test.espresso.matcher.ViewMatchers.withParent;
+//import static android.support.test.espresso.matcher.ViewMatchers.withText;
+//import static org.hamcrest.Matchers.allOf;
+//import static org.hamcrest.Matchers.is;
```

Comment:
android imports should be above third party and java/javax imports

Reply:
I will make those changes now